addons/vehicles/functions/fnc_startEngine.sqf

```diff
@@ -20,6 +20,8 @@ params ["_vehicle", "_isEngineOn"];
 
 if (!_isEngineOn || {floor abs speed _vehicle > 0 || {!isNull isVehicleCargo _vehicle}}) exitWith {};
 
+private _startupDelay = _vehicle getVariable [QGVAR(engineStartDelay), getNumber (configOf _vehicle >> QGVAR(engineStartDelay))];
+
```

Comment:
```suggestion
if (_startupDelay <= 0) exitWith {};

```

Reply:
Why 0 not allowed? What if someone want play spaceships

Reply:
this is just to avoid adding and removing the `CBA_fnc_addPerFrameHandler`
if the delay is 0 it shouldn't do anything anyway